core/client/src/main/java/alluxio/client/block/UnderStoreBlockInStream.java

```diff
@@ -85,7 +93,7 @@ public long remaining() {
   public void seek(long pos) throws IOException {
     long offset = mPos - mInitPos;
     if (pos < offset) {
-      setUnderStoreStream(pos);
+      setUnderStoreStream(mInitPos + pos);
```

Comment:
I just want to clarify. Are you saying you prefer to have setUnderStoreStream() take in a relative pos, instead of the absolute pos (existing implementation)?